controllers/expression.js

```diff
@@ -53,8 +53,8 @@ class ExpressionController {
 
   async deleteExpressionFromUser(request, response) {
     try {
-      const { expressionId } = request.body;
-      const { userId } = request.user;
+      const { expressionId } = request.params;
```

Comment:
I see that in `routes/expressions.js:16` the route is `/:id` but here you are taking from params `expressionId`.